boa3_test/tests/test_interop.py

```diff
@@ -619,6 +619,98 @@ def test_hash256_str(self):
         output = Boa3.compile(path)
         self.assertEqual(expected_output, output)
 
+    def test_base64_encode(self):
+        expected_output = (
+            Opcode.INITSLOT
+            + b'\x00\x01'
+            + Opcode.LDARG0
+            + Opcode.SYSCALL
+            + Interop.Base64Encode.interop_method_hash
+            + Opcode.RET
+        )
+
+        path = '%s/boa3_test/test_sc/interop_test/Base64Encode.py' % self.dirname
+        output = Boa3.compile(path)
+        self.assertEqual(expected_output, output)
+
+        import base64
+        engine = TestEngine(self.dirname)
+        expected_result = base64.b64encode(b'unit test')
+        if isinstance(expected_result, bytes):
```

Comment:
Instead of converting the expected_result to str, I think it's better to convert the result.
Because the result of b64encode is always bytes, but the conversion of ByteString in the TestEngine can be both str and bytes.

This conversion exists in some tests because if the types are different, the assertion will fail